k8s/spack/spackbot-spack-io/secrets-dummy.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: spack-bot-credentials
+  namespace: spack
+  labels:
+    app: spackbot-spack-io
+    svc: web
+data:
+  gitlab_token: |
```

Comment:
Ah gotcha on how this works! Where were the secrets coming from before this file (for the webhook, etc). We’re they just set on the command line but not auto renewed or generated?

Reply:
I had a file like this that wasn't checked in (b/c secrets) and got lost in the move from `kubernetes` to `k8s`.  I deleted the old `kubernetes` directory where it lived. So I checked this one in and I figure why not use the magic formatting commands as the dummy secret placeholder values.